tests/integration/components/resources-list/edit-resource/component-test.ts

```diff
@@ -0,0 +1,132 @@
+import { render, click as untrackedClicked, click } from '@ember/test-helpers';
```

Comment:
I didn't know you could do this... Does this just import `click` under two separate names? Let's use the `click` helper from `ember-osf-web/tests/helpers`